### PR TITLE
Hypospray belt fixes.

### PR DESCRIPTION
**Pull request by Mrrpip**
## About The Pull Request

Re-adds sprites and storage functionality to the hypospray belt.

https://media.discordapp.net/attachments/500801682058379265/991489041491054692/unknown.png

## Why It's Good For The Game

Well. A non-broken belt that can hold it's own stuff and has sprites is good. I think.

## Changelog
:cl:
fix: Fixed a bunch of broken crap from the hypospray belt.
/:cl:

### CODE REVIEW

**Review by TiviPlus — APPROVED** [review]

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -231,6 +233,7 @@
 	new /obj/item/reagent_containers/hypospray/autoinjector/spaceacillin(src)
 	new /obj/item/reagent_containers/hypospray/autoinjector/hypervene(src)
 	new /obj/item/reagent_containers/hypospray/autoinjector/quickclot(src)
+	new /obj/item/reagent_containers/hypospray/autoinjector/peridaxon_plus(src)
```
Comment (TiviPlus) [inline code/game/objects/items/storage/belt.dm:236]: Holdon whats this this isnt a fix

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by TiviPlus** [comment]
Also those sprites are practically garaunteed to get denied by Kuro

File: code/game/objects/items/storage/belt.dm (hunk 1 above)
Comment (Mrrpip) [reply code/game/objects/items/storage/belt.dm:236]: This was the original state of the belt, when peridaxon was removed, a standard peridaxon injector was too, having been replaced with a QC injector.

**Comment by Mrrpip** [comment]
> Also those sprites are practically garaunteed to get denied by Kuro

Not my place to say. They were merged originally.
Meaning either Kuro agreed, wasn't asked or flat out ignored.

I was asked to fix the missing sprites, so I did.

File: code/game/objects/items/storage/belt.dm (hunk 1 above)
Comment (Mrrpip) [reply code/game/objects/items/storage/belt.dm:236]: As for 'Peridaxon+ is rare'...You can just mab 50 phoron sheets from the engine room.

**Review by Kuro020 — APPROVED** [review]

File: code/game/objects/items/storage/belt.dm (hunk 1 above)
Comment (BraveMole) [reply code/game/objects/items/storage/belt.dm:236]: Still don't understand how it's a fix ?

**Pull request closed without merging**